saleor/graphql/account/utils.py

```diff
@@ -173,3 +173,133 @@ def get_groups_which_user_can_manage(user: "User") -> List[Optional[Group]]:
             editable_groups.append(group)
 
     return editable_groups
+
+
+def get_not_manageable_permissions_after_group_deleting(group):
+    """Return permissions that cannot be managed after deleting the group.
+
+    After removing group, for each permission, there should be at least one staff member
+    who can manage it (has both “manage staff” and this permission).
+    """
+    group_pk = group.pk
+    groups_data = get_group_to_permissions_and_users_mapping()
+    not_manageable_permissions = set(groups_data[group_pk]["permissions"])
+
+    # get users from groups with manage staff and look for not_manageable_permissions
+    # if any of not_manageable_permissions is found it is removed from set
+    manage_staff_users = get_users_and_look_for_permissions_in_groups_with_manage_staff(
+        group_pk, groups_data, not_manageable_permissions
```

Comment:
`group_pk` is passed to this function only to skip this particular PK when iterating over `groups_data`. What if we did something like:

```
not_manageable_permissions = set(groups_data.pop(group_pk)["permissions"])
```

Then we would have one less parameter in `get_users_and_look_for_permissions_in_groups_with_manage_staff` and in `look_for_permission_in_users_with_manage_staff` plus we wouldn't have to do the `if pk == group_pk:` skipping in both functions.

Reply:
Yes, I changed it in #5438